wire-ios-request-strategy/Sources/Request Strategies/Conversation/ConversationService.swift

```diff
@@ -247,6 +263,7 @@ public final class ConversationService: ConversationServiceInterface {
             unqualifiedUserIDs = usersExcludingSelfUser.compactMap(\.remoteIdentifier)
         }
 
+        createGroupFlow.checkpoint(description: "sending request to backend start")
```

Comment:
Don't we log all networks request? Isn't it a bit redundant to log this as well.

Reply:
yes we do, but when filtering on this flow, the network logs will not show.
I wanted to know when it started. Maybe there's a way to show it without this checkpoint

Reply:
removed for now